src/helm/benchmark/model_deployment_registry.py

```diff
@@ -63,27 +63,31 @@ class ModelDeployments:
 _name_to_model_deployment: Dict[str, ModelDeployment] = {}
 
 
+def register_model_deployment(model_deployment: ModelDeployment) -> None:
+    hlog(f"Registered model deployment {model_deployment.name}")
+    _name_to_model_deployment[model_deployment.name] = model_deployment
+
+    # Auto-register a model with this name if none exists
+    model_name = model_deployment.model_name or model_deployment.name
+    if model_name not in MODEL_NAME_TO_MODEL:
+        model = Model(
+            group="unknown",
+            name=model_name,
+            tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG],
+        )
+        MODEL_NAME_TO_MODEL[model_name] = model
+        ALL_MODELS.append(model)
+        hlog(f"Registered default metadata for model {model_name}")
```

Comment:
I am guessing this is what is registering the two removed models?

Reply:
This is for registering models from the flag.